core/src/test/java/io/questdb/cairo/CairoEngineTest.java

```diff
@@ -515,4 +517,18 @@ private void createY() {
             CairoTestUtils.create(model);
         }
     }
+
+    @Test
+    public void testDuplicateTableCreation() throws Exception {
+        try (TableModel model = new TableModel(configuration, "x", PartitionBy.NONE)
```

Comment:
I'm not sure whether this should be wrapped by this `assertLeak()` thingies. not sure what are the rules to follow.

Reply:
yeah, TestUtils.assertLeak would help here

Reply:
done. and it actually found a real leak!